notebooks/CSSP_20CRDS_Tutorials/tutorial_2_data_preparation.ipynb

```diff
@@ -373,7 +373,21 @@
     "\n",
     "To make your analysis faster and easier, you can extract a smaller part of the model domain. In these examples we will work with Shanghai, but you can choose any region you want. \n",
     "\n",
-    "<b>Note:</b> The original model data is on a rotated pole grid system and we need to the extract_rot_method() in order to get the data on to a regular grid system"
+    "<b>Note:</b> The original model data is on a rotated pole grid system which as seen in this diagram its coordinates are relative to the standard pole so we need to first get our data on to a standard grid system. We use the *extract_rot_method()* to do this."
```

Comment:
I suggest: 
The original model data is on a rotated pole grid system, as show in the diagrams below. The x and y coordinates are not true latitude and longitude so to extract a latitude-longitude box, we use the *extract_rot_cube()* function to do this. It works by first calculating the true latitude and longitue of each grid cell and uses these to select which are in the area of interest.